CIP6/CIP6.md

```diff
@@ -0,0 +1,885 @@
+
+## Specification
+
+#### On Chain referenced metadata file
+We define one more line for the on chain referenced metadata file that references another json file on a url with the extended metadata.  The proposed metadata is as follows:
+
+| key           | Value                                | Rules  |
+| ---           | ---                                  | ---  |
+|  `ticker`       | Pool ticker.  uppercase              | 5 Characters Maximum, Uppercase letters and numbers  |
+|  `description` | Pool Description.  Text that describes the pool | 50 Characters Maximum |
+|  `homepage` | A website URL for the pool| 64 Characters Maximum, must be a valid URL |
+|  `name` | A name for the pool | 50 Characters Maximum |
+|  `extended` | A url for extended metadata| Optional, 64 Characters Maximum, must be a valid URL |
+
+#### Extended Metadata
+The file located at the URL for extended data is a json compliant text file with the following top level fields:
+
+| key           | Description                                |
+| ---           | ---                                  |
+| `serial` | set to YYYYMMDDxx on every update |
+|  `itn` | ITN pool validation data |
+|  `info` | Corporate image, contacts and about details |
+| `pools` | Pool details |
+
+Since this extended metadata file has no checksum and can be updated at any time by the pool operator, a **serial number** is useful to easily identify updates.
+
+The purpose of this first (**itn**) extension is the verifiable linking of an ITN pool ticker with its counterpart in Mainnet to identify fraudulent duplicates.
+
+#### Complete Extended Metadata Example
+```
+{
+    "serial": 2020072001,
+    "itn": {
+        "owner": "ed25519_pk1...",
+        "witness": "ed25519_sig1..."
+    },
+    "info": {
+        "CI": {
+            "logo_url_png_icon_64x64": "https://mycoolpool.com/icon.png",
+            "logo_url_png_logo": "https://mycoolpool.com/logo.png",
+            "color_main": "#RRGGBB",
+            "logo_vector": "<?xml version='1.0' ?><!DOCTYPE svg  PUBLIC '-//W3C//DTD SVG 1.1//EN'  'http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd'><svg enable-background='new 0 0 24 24' height='24px' id='Layer_1' version='1.1' viewBox='0 0 24 24' width='24px' xml:space='preserve' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink'><g><path d='M12,0C5.4,0,0,5.4,0,12s5.4,12,12,12s12-5.4,12-12S18.6,0,12,0z M12,22C6.5,22,2,17.5,2,12S6.5,2,12,2s10,4.5,10,10   S17.5,22,12,22z'/><path d='M12,9c-1.1,0-2,0.9-2,2c0,0.6,0.3,1.1,0.7,1.5C10.3,12.9,10,13.4,10,14c0,1.1,0.9,2,2,2s2-0.9,2-2c0-0.6-0.3-1.1-0.7-1.5   c0.4-0.4,0.7-0.9,0.7-1.5C14,9.9,13.1,9,12,9z M12,15c-0.6,0-1-0.4-1-1s0.4-1,1-1s1,0.4,1,1S12.6,15,12,15z M12,12   c-0.6,0-1-0.4-1-1s0.4-1,1-1s1,0.4,1,1S12.6,12,12,12z'/><path d='M12,6c-3.3,0-6,2.7-6,6s2.7,6,6,6s6-2.7,6-6S15.3,6,12,6z M12,17c-2.8,0-5-2.2-5-5s2.2-5,5-5s5,2.2,5,5S14.8,17,12,17z'/></g></svg>"
```

Comment:
I don't think it's wise to enable XML embedding, while there are some benefits (like reducing number of connections required to fetch the resources) I think the SVG should be external resource just like the PNG, SVG files can be also large, and XML in general opens whole can of worms in terms of CVEs.

Reply:
good point. 
While brainstorming for the CIP our goal was not really to find definitive field and data types, but throw a whole bunch of stuff on the table, in order to give a picture of what it could be used for. 

I personally see a vector graph as an _alternative_ to pixel graphics, in case we expect extensive byte-size and additional requests, resulting in chaotically colorful logo walls. Or might a  reference to a classic multisize favicon file is all we need.

Reply:
@gufmar lets plan to replace this with a link to an svg then?

Reply:
removed